src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
is better
 html {
  box-sizing: border-box;
}
*,
*::before,
*::after {
  box-sizing: inherit;
}